apps/studio/components/interfaces/Storage/Storage.constants.ts

```diff
@@ -56,3 +59,31 @@ export const CONTEXT_MENU_KEYS = {
   STORAGE_ITEM: 'STORAGE_ITEM',
   STORAGE_FOLDER: 'STORAGE_FOLDER',
 }
+
+// New bucket types configuration
+export interface BucketTypeConfig {
+  displayName: string
+  description: string
+  docsUrl: string
+}
+
+export const BUCKET_TYPES = {
+  media: {
+    displayName: 'Media',
+    description: 'General file storage for most types of digital content.',
+    docsUrl: `${DOCS_URL}/guides/storage/buckets/fundamentals`,
+  },
+  analytics: {
+    displayName: 'Analytics',
+    description: 'Purpose-built storage for analytical workloads.',
+    docsUrl: `${DOCS_URL}/guides/storage/analytics/introduction`,
+  },
+  vectors: {
+    displayName: 'Vectors',
+    description: 'Purpose-built storage for vector data.',
+    docsUrl: `${DOCS_URL}/guides/storage/vectors`,
+  },
+} as const satisfies Record<string, BucketTypeConfig>
```

Comment:
do we need this actually? BUCKET_TYPES would already be inferred correctly even without this declaration
![Uploading image.png…]()